app/screens/navigation.ts

```diff
@@ -695,7 +695,21 @@ export function setButtons(componentId: string, buttons: NavButtons = {leftButto
     mergeNavigationOptions(componentId, options);
 }
 
-export function showOverlay(name: string, passProps = {}, options: Options = {}) {
+export function setTitle(componentId: string, title: string) {
```

Comment:
Thid is definitely not needed

Reply:
Again, this is not needed, in any case this can be called directly in the screen component

Reply:
Checking this

Reply:
Updated